gorgone/gorgone/modules/core/action/class.pm

```diff
@@ -550,7 +550,7 @@ sub action_command {
 
 sub action_processcopy {
     my ($self, %options) = @_;
-    
+    my $dest_filename = $options{data}->{content}->{destination};
```

Comment:
Why is there no check whether `$options{data}->{content}->{destination}` has a value? Neither is there on line 601.